pages/order.vue

```diff
@@ -5,21 +5,66 @@
         </div>
         <div v-else>
           <h2 class="section-title">Cool!  Let's get some information and get you started.</h2>
-
           <div class="step">
-            <font-awesome-icon icon="check" /> &nbsp;You've selected the {{plan.name}} package, with {{planTerm}}ly payments of {{currentPrice | currency}}.
-            <br>
-            <div v-if="planTerm == 'month'">
-              <a @click="setTerm('year')">Save {{yearlySavings}}% by switching to yearly payments ({{yearlyPrice | currency}}/year)</a>
-            </div>
-            <div v-else>
-              <a @click="setTerm('month')">Switch back to monthly payments ({{monthlyPrice | currency}}/month)</a>
-            </div>
+            <template v-if="!isTeamPackageInterfaceEnabled">
+              <font-awesome-icon icon="check" /> &nbsp;You've selected the {{plan.name}} package, with {{planTerm}}ly payments of {{currentPrice | currency}}.
+              <br>
+              <div v-if="planTerm == 'month'">
+                <a @click="setTerm('year')">Save {{yearlySavings}}% by switching to yearly payments ({{yearlyPrice | currency}}/year)</a>
+              </div>
+              <div v-else>
+                <a @click="setTerm('month')">Switch back to monthly payments ({{monthlyPrice | currency}}/month)</a>
+              </div>
+              or
+              <div>
+                <a @click="enableTeamPackageInterface">Purchase for a team and save up to {{teamPurchaseMaxSavings}}%</a>
+              </div>
+            </template>
+            <template v-else>
+              <div class="headline">You've selected the {{plan.name}} package</div>
+              <v-simple-table>
+                <template v-slot:default>
+                  <thead>
+                  <tr>
+                    <th class="text-left">Number users</th>
+                    <th class="text-left">Cost per month</th>
+                    <th class="text-left">Cost per year</th>
+                  </tr>
+                  </thead>
+                  <tbody>
+                  <tr v-for="range in usersRangeKeys" :key="range">
+                    <td><v-icon>{{getIcon(range)}}</v-icon> {{ range }}</td>
+                    <td>{{ plan['month'].prices[range] | currency }}</td>
+                    <td>{{ plan['year'].prices[range] | currency }}</td>
+                  </tr>
+                  </tbody>
+                </template>
+              </v-simple-table>
+              <NumberInput label="Please select the number of seats:" :min="1" :value="seats" @input="setSeats" />
+              <v-radio-group :value="planTerm" @change="$event => setTerm($event)">
+                <template v-slot:label>
```

Comment:
I prefer using the shortcut `#label` vs `v-slot:label`.

Reply:
Funny, I didn't know about that syntax. Thank you.